DEPS

```diff
@@ -21,8 +21,9 @@ vars = {
   'chromium_git': 'https://chromium.googlesource.com',
   'dart_git': 'https://dart.googlesource.com',
   'fuchsia_git': 'https://fuchsia.googlesource.com',
-  'skia_git': 'https://skia.googlesource.com',
   'github_git': 'https://github.com',
+  'libwebp_git': 'https://chromium.googlesource.com/webm',
```

Comment:
`https://chromium.googlesource.com` is already present in the `var` above. Please just use that var directly below.